src/components/input/Input.tsx

```diff
@@ -16,7 +16,7 @@ export function inputStyles({ classes, feedback }: InputStylesOptions) {
   return classnames(
     'focus-visible-ring ring-inset border rounded w-full p-2',
     'bg-grey-0 focus:bg-white disabled:bg-grey-1',
-    'placeholder:text-color-grey-5 disabled:placeholder:color-grey-6',
+    'placeholder:text-grey-6 disabled:placeholder:text-grey-7',
```

Comment:
Neither `text-color-grey-5` or `color-grey-6` were valid tailwind classes, but even if they were, `grey-5` is too light in contrast with white.

Reply:
We really ought to have a lint to check validity of classes. I suspect there was confusion here because we do have some custom [`text-color-*` properties](https://github.com/hypothesis/frontend-shared/blob/9f84354fdd7541e93664c41422b6b0989fca3aad/src/tailwind.preset.js#L67).

Reply:
> I suspect there was confusion here because we do have some custom [text-color-* properties](https://github.com/hypothesis/frontend-shared/blob/9f84354fdd7541e93664c41422b6b0989fca3aad/src/tailwind.preset.js#L67).
yep, that was also my thinking.